### DISCUSSION TITLE
EQ4 MyOpenMath

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#332] [General]
Reviewer: @tdegeorge

**Comment by tdegeorge** [comment]
Do we want a question where you have an absolute value equal to another absolute value? Like in question 5:
<img width="150" alt="Screenshot 2024-07-03 at 1 36 20 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/143553140/ecd5dfc6-be95-46a6-bdfd-a14f5add470d">

**Comment by tdegeorge** [comment]
I think all the others looks good. It has a good mix of different types of answers and questions.
